docs/xocto/numbers.md

```diff
@@ -0,0 +1,16 @@
+# Numbers
+
+## One-stop shop for all things numbers
+
+The `numbers` module is intended as your one-stop shop for all things numbers.
```

Comment:
Nitpick, nonblocking: This is a touch _too_ general - "all things numbers" is a lot of things!

```suggestion
The `numbers` module is used to manipulate numbers, typically by rounding and simplifying them, in utilities-industry-standard ways.
```

Reply:
I agree 100%